Configurations/CommonBase.xcconfig

```diff
@@ -74,7 +74,7 @@ OTHER_LDFLAGS = $(inherited) $(WK_COMMON_OTHER_LDFLAGS);
 WK_COMMON_OTHER_TAPI_FLAGS = -x objective-c++ -std=c++2b -fno-rtti $(WK_SANITIZER_OTHER_TAPI_FLAGS);
 OTHER_TAPI_FLAGS = $(inherited) $(WK_COMMON_OTHER_TAPI_FLAGS);
 
-WK_DEFAULT_WK_AUDIT_SPI[sdk=iphoneos*] = $(WK_NOT_$(WK_ANY_SANITIZER_ENABLED));
+WK_DEFAULT_WK_AUDIT_SPI[sdk=iphoneos*] = $(WK_NOT_$(WK_OR_$(WK_ANY_SANITIZER_ENABLED)_$(OVERRIDE_SUPPORTS_TEXT_BASED_API)));
```

Comment:
Actually, this doesn't work because `Base.xcconfig` and therefore this is included before `LocalOverrides.xcconfig` for each of the Debug/Release Configurations... I'm not sure if changing the include order would be appropriate. It doesn't seem totally insane since we probably want to be able to override most things including shared logic. On the other hand, we could also just have folks set `WK_AUDIT_SPI` in the LocalOverrides...

@emw-apple thoughts?